docs/api/vega-wallet/v2-api/integrate-wallet-service.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Integrate the wallet service
+hide_title: false
+sidebar_position: 5 
+---
+
+This guide will walk you through how to set up the Vega  Wallet from scratch to use when building a Vega dApp, or integrating with CI/CD. 
+
+You can find the current working CI/CD implementation in the [front-end GitHub repo ↗](https://github.com/vegaprotocol/frontend-monorepo/blob/develop/.github/actions/setup-vegawallet/action.yml).
+
+Depending on your end-goal, you may be able to skip some steps. API token-related steps are only needed for end-to-end and CI/CD integration. Also, if you already have Vega Wallet prepared and configured, you may only need the last step to run the wallet.
+
+1. **Create two files for your `passphrase` and `recovery` phrase** - `passphrase` will be used as a wallet and API token password (you may want to use two different ones), and `recovery` will contain recovery phrase of the wallet you want to use (you can skip it if you don’t have a wallet yet, and want to create a new one).
+
+2. **Download and install the Vega Wallet software.** 
+    There are two ways you can do that:
+    a. Download the latest release of the [Vega software ↗](https://github.com/vegaprotocol/vega/releases) and unzip it to a shared location. To test if it worked, run the `vega version` command.
+    b. Clone the [Vega repo ↗](https://github.com/vegaprotocol/vega). Make sure you have installed `go`. Read the guide for [getting started with Go for Vega ↗](https://github.com/vegaprotocol/vega/blob/develop/GETTING_STARTED.md#installing-golang) if you need help. Set up the `GOBIN` environment and add it to `PATH` (e.g. `export GOBIN="$HOME/go/bin; export PATH=$GOBIN/:$PATH`. Run `go install ./…` to install the binaries. Verify by running the command `vega version`.
```

Comment:
Go install ./cmd/vegawallet ./cmd/vega

Reply:
@ValentinTrinque does not `go install ./...` do the same?

Reply:
I shall verify but I am not sure.

Reply:
YOu are right, it's properly resolving the project mains.